corehq/apps/users/bulk_download.py

```diff
@@ -64,7 +64,7 @@ def _get_devices(user):
             user.devices, key=lambda d: d.last_used, reverse=True
         )])
 
-    def _make_user_dict(user, group_names, location_cache):
+    def _make_user_dict(user, group_names, location_cache, domain):
```

Comment:
Nitpick: maybe rename one of the `domain` variables? It's a bit confusing to have both this one and the "outer" one passed to `parse_users`.